src/Resources/config/cache.xml

```diff
@@ -70,15 +70,5 @@
         <service id="sonata.cache.invalidation.simple" class="Sonata\Cache\Invalidation\SimpleCacheInvalidation">
             <argument type="service" id="logger"/>
         </service>
-        <service id="sonata.cache.command.flush" class="Sonata\CacheBundle\Command\CacheFlushCommand">
-            <argument type="constant">null</argument>
-            <argument type="service" id="sonata.cache.manager"/>
-            <tag name="console.command"/>
-        </service>
-        <service id="sonata.cache.command.flushall" class="Sonata\CacheBundle\Command\CacheFlushAllCommand">
-            <argument type="constant">null</argument>
-            <argument type="service" id="sonata.cache.manager"/>
-            <tag name="console.command"/>
-        </service>
```

Comment:
This looks like a BC-break for any piece of code relying on the existence of `sonata.cache.command.flushall`… maybe just remove the tag? The removal should probably happen on next major release.

Reply:
I created aliases, to make sure it is actually backwards compatible. Thanks for that hint!